libs/openblas/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=OpenBLAS
+PKG_SOURCE_VERSION:=e55ec82bb92338d09ecd77357da3fcdfac0a7902
+PKG_VERSION:=0.3.9-$(PKG_SOURCE_VERSION)
+PKG_RELEASE:=1
+#PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+#PKG_SOURCE_URL:=https://github.com/xianyi/OpenBLAS/archive/
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2020-04-19
+PKG_SOURCE_URL:=https://github.com/martin-frbg/OpenBLAS.git
```

Comment:
it's a good idea to point to release tarballs, unless impossible;
from what i can tell, that should be doable for this repo;
but i am a bit confused about the official tarball for this package;
you are pointing to `https://github.com/martin-frbg/OpenBLAS.git` officially it's another repo;

if you need patches from that repo, pick them out and put them in the `patches` sub-folder of the package;

Reply:
Cause I was working on a specific PR to test in `docker run --rm -it openwrtorg/sdk:ramips-mt76x8-19.07.2`.
See https://github.com/xianyi/OpenBLAS/pull/2565